conf/nginx.conf.template

```diff
@@ -41,8 +41,8 @@ http {
     # Logging Settings
     ##
 
-    access_log ${LOG_DIR}/access.log;
-    error_log ${LOG_DIR}/error.log;
+    ${ACCESS_LOG_DIRECTIVE}
```

Comment:
Will be `access_log ${LOG_DIR}/access.log;` when `ENABLE_ACCESS_LOGS=true`.

Will be `access_log off;` when `ENABLE_ACCESS_LOGS` is not set (default).